### PR TITLE
[fix](chore) fix env.sh build on macOS

### PR DESCRIPTION
**Pull request by zy-kkk**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by pingchunzhang** [comment]
LGTM

**Review by pingchunzhang — APPROVED** [review]
LGTM

**Comment by zy-kkk** [comment]
run buildall

**Review by hf200012 — APPROVED** [review]

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 46.76 seconds
 stream load tsv:          548 seconds loaded 74807831229 Bytes, about 130 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.3 seconds inserted 10000000 Rows, about 341K ops/s
 storage size: 17162184717 Bytes

File: env.sh [hunk 1]
```diff
@@ -146,7 +146,7 @@ elif [[ "${DORIS_TOOLCHAIN}" == "clang" ]]; then
     fi
 
     covs=()
-    while IFS='' read -r line; do covs+=("${line}"); done < <(find "${DORIS_CLANG_HOME}" -name "llvm-cov*")
+    while IFS='' read -r line; do covs+=("${line}"); done <<<"$(find "${DORIS_CLANG_HOME}" -name "llvm-cov*")"
```
Comment (adonis0147) [inline env.sh:149]: What is wrong with the original statement?

**Pull request merged**